Packs/Venafi/Integrations/README.md

```diff
@@ -0,0 +1,217 @@
+| StageGreater | Certificates with a stage greater than the specified stage (does not include specified stage). | Optional | 
+| StageLess | Certificates a stage less than the specified stage. | Optional | 
+| ValidationDisabled | Include only certificates with validation disabled (1) or enabled (0). | Optional | 
+| C | Country attribute of Subject DN. | Optional | 
+| CN | Common name attribute of Subject DN. | Optional | 
+| Issuer | Issuer DN. Note, since most Issuer DNs include commas between DN components, it is important to surround the complete Issuer DN within double quotes (“). In addition, if the Issuer DN includes double quotes, each double quote should be prefixed by another double quote. | Optional | 
+| KeyAlgorithm | Algorithm for the public key in the certificate (e.g., RSA, DSA). | Optional | 
+| KeySize | Size of the public key in the certificate (e.g., 2048). Accepts multiple comma separated values. | Optional | 
+| KeySizeGreater | Key size greater than the specified value. | Optional | 
+| KeySizeLess | Key size less than the specified value. | Optional | 
+| L | Locality/City attribute of Subject DN in certificates. | Optional | 
+| O | Organization attribute of Subject DN in certificates. | Optional | 
+| S | State/Province attribute of Subject DN in certificates. | Optional | 
+| Serial | Serial number of the certificate. | Optional | 
+| SignatureAlgorithm | The algorithm used to sign the certificate (e.g. SHA1RSA). | Optional | 
+| ValidFrom | Date on which the certificate was issued (e.g., 2015- 10-08T19:15:35.6431456Z or 2015-10-08). | Optional | 
+| ValidTo | Date on which the certificate expires (e.g., 2015-10- 08T19:15:35.6431456Z or 2015-10-08). | Optional | 
+| ValidToGreater | Certificates that expire after the specified date. | Optional | 
+| ValidToLess | Certificates that expire before the specified date. | Optional | 
+| Limit | The maximum number of certificates to return. | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Venafi.Certificate.CreatedOn | date | Certificate creation date | 
+| Venafi.Certificate.DN | string | Certificate DN | 
+| Venafi.Certificate.Name | string | Certificate name | 
+| Venafi.Certificate.ParentDN | string | Certificate parent DN | 
+| Venafi.Certificate.SchemaClass | string | Certificate schema | 
+| Venafi.Certificate.ID | string | Certificate ID \(GUID\) | 
+
+
+#### Command Example
+```!venafi-get-certificates Limit="1"```
+
+#### Context Example
+```json
+{
+    "Venafi": {
+        "Certificate": {
+            "CreatedOn": "2018-07-16T16:35:35.9468326Z",
+            "DN": "\\VED\\Policy\\Venafi Operational Certificates\\WIN-MLK71Q10559",
+            "ID": "2a25573b-745c-4018-806a-e5c73f424675",
+            "Name": "WIN-MLK71Q10559",
+            "ParentDN": "\\VED\\Policy\\Venafi Operational Certificates",
+            "SchemaClass": "X509 Server Certificate"
+        },
+        "Certificats": {
```

Comment:
should we remove the deprecated context paths from the README?

Reply:
removed